src/components/Layout/Header/Header.tsx

```diff
@@ -190,6 +190,11 @@ const Header: React.FC<HeaderProps> = ({
                 </div>
               </NavListLink>
             </NavListItem>
+            <NavListItem>
+              <NavLink activeClassName="active" to="/careers">
```

Comment:
We should create an issue to centralise the routes to avoid dead links in the future.

Reply:
@DiogoSoaress, I created on [here](https://github.com/gnosis/safe-landing-page/issues/45).